bluesky/utils.py

```diff
@@ -742,7 +743,7 @@ def all_safe_rewind(devices):
     return True
 
 
-class PersistentDict(zict.Func):
+class PersistentDict(collections.abc.MutableMapping):
```

Comment:
I like this much better. It feels more canonical if that makes sense.

The only thing that I could find, which worked before, but will no longer work as a result of this, is:

```python
p = PersistentDict(...)
p.close()
```

Not sure we ever used that though, or would even want it to work. It would close the `self._file`, putting the PersistentDict into an awkward state.